charts/external-nodes/templates/scripts-cm.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
rename filename to scripts-configmap.yaml

Reply:
i think -cm is good enough:

kubectl get cm - so it is formal
all our cf-helm charts uses this naming